cmd/collection/main.go

```diff
@@ -115,7 +116,9 @@ func main() {
 			"how many additional cluster members we propagate transactions to")
 		flags.UintVar(&builderExpiryBuffer, "builder-expiry-buffer", builder.DefaultExpiryBuffer,
 			"expiry buffer for transactions in proposed collections")
-		flags.Float64Var(&builderPayerRateLimit, "builder-rate-limit", builder.DefaultMaxPayerTransactionRate, // no rate limiting
+		flags.BoolVar(&builderPayerRateLimitDryRun, "builder-rate-limit-dry-run", true, // dry-run rate limiting
+			"determines whether rate limit configuration should be enforced (false), or only logged (true)")
+		flags.Float64Var(&builderPayerRateLimit, "builder-rate-limit", 1.0, // 1 tx/collection
```

Comment:
Is this flag temporary and for debugging purpose?

I'm a bit concerned about changing the default value. 

I would have to remember that when upgrading to the new version, the default setting for rate limiting is changed from no rate limit to 1 tx/collection. 

Is this intended?